wallet-new/server/Cardano/Wallet/API/V1/Swagger.hs

```diff
@@ -231,150 +181,223 @@ The number of entries to display for each page. The minimum is **1**, whereas th
+
+Software Version | Git Revision
+-----------------|-------------------
+$deGitRevision   | $deSoftwareVersion
+|]
 
-instance ToDocs WalletSoftwareUpdate where
-  descriptionFor _ = "A programmed update to the system."
 
-instance ToDocs NodeSettings where
-  descriptionFor _ = "A collection of static settings for this wallet node."
+-- | Provide additional insights on V1 documentation
+highLevelDescription :: DescriptionEnvironment -> T.Text
+highLevelDescription DescriptionEnvironment{..} = [text|
+This is the specification for the Cardano Wallet API, automatically generated as a [Swagger](https://swagger.io/)
+spec from the [Servant](http://haskell-servant.readthedocs.io/en/stable/) API of [Cardano](https://github.com/input-output-hk/cardano-sl).
 
-instance ToDocs BlockchainHeight where
-  descriptionFor _ = "The height of the blockchain."
+Software Version | Git Revision
+-----------------|-------------------
+$deGitRevision   | $deSoftwareVersion
 
-instance ToDocs SyncProgress where
-  descriptionFor _ = "The sync progress with the blockchain."
+Getting Started
+---------------
 
-instance ToDocs SlotDuration where
-  descriptionFor _ = "The duration for a slot."
+In the following examples, we will use *curl* to illustrate request to an API running on the
+default port **8090**.
 
-instance ToDocs LocalTimeDifference where
-  descriptionFor _ = "The time difference between this node clock and the NTP server."
+> Please note that wallet web API uses TLS for secure communication. Requests to the API need
+to send a client CA certificate that was used when launching the node and identifies the client as
+being permitted to invoke the server API.
 
-instance ToDocs NodeInfo where
-  descriptionFor _ = "A collection of dynamic information for this wallet node."
+Creating a New Wallet
+=====================
 
-instance ToDocs WalletStateSnapshot where
-  descriptionFor _ = "Dump current wallet state"
+You can create your first wallet using the `POST /api/v1/wallets` endpoint as follow:
 
-instance ToDocs (V1 InputSelectionPolicy) where
-  descriptionFor _ = "A policy to be passed to each new `Payment` request to "
-                  <> "determine how a `Transaction` is assembled. "
-                  <> "Possible values: [" <> possibleValuesOf @(V1 InputSelectionPolicy) Proxy <> "]."
+```
+curl -X POST https://localhost:8090/api/v1/wallets                     \
+     -H "Content-Type: application/json; charset=utf-8"                \
+     -H "Accept: application/json; charset=utf-8"                      \
+     --cacert ./scripts/tls-files/ca.crt                               \
+     -d '{                                                             \
+  "operation": "create",                                               \
+  "backupPhrase": ["squirrel", "material", "silly", "twice", "direct", \
+    "slush", "pistol", "razor", "become", "junk", "kingdom", "flee"],  \
+  "assuranceLevel": "normal",                                          \
```

Comment:
A note we should add (just recalled a conversation I had with @jmitchell ): We need to make clear that those 12 words mnemonic are given just as an example and that is not secure by any means reusing it for a production wallet, and that doing so might cause funds to be lost.

Reply:
Good one. I'll drop a sentence early in the guide and maybe elaborate a bit more in the part about `Mnemonic Codes` :+1: